winit-win32/src/event_loop.rs

```diff
@@ -1833,20 +1753,17 @@ unsafe fn public_window_callback_inner(
                 primary: true,
                 state: Pressed,
                 position,
-                button: match xbutton {
-                    1 => Back,
-                    2 => Forward,
-                    _ => Other(xbutton),
-                }
-                .into(),
+                // 1 is defined as back, 2 as forward; other codes are unexpected.
+                button: MouseButton(xbutton as u8 + MouseButton::BACK.0 - 1).into(),
```

Comment:
It's well documented exactly which codes are produced here, and my testing agreed: this code should only ever yield `MouseButton::BACK` and `FORWARD`.